JSON/game.json

```diff
@@ -0,0 +1,22 @@
+{
+  "Games": [
+    {
+      "multiplayer": "true",
+      "last_played_at": "2019-01-01",
+      "publish_date": "2020-01-01",
+      "id": 465
+    },
+    {
+      "multiplayer": "Yes",
+      "last_played_at": "2023-05-03",
+      "publish_date": "2023-05-03",
+      "id": 780
+    },
+    {
+      "multiplayer": "yes",
+      "last_played_at": "2023-05-03",
+      "publish_date": "2023-05-03",
+      "id": 320
+    }
+  ]
+}
```

Comment:
As suggested by the previous code reviewerWhen creating a book there is no way to add the label "title and color" which means the label can't be saved in the label.json if can't create it, kindly fix that